Source/problems/bc_ext_fill_nd.F90

```diff
@@ -76,20 +76,13 @@ subroutine ext_fill(lo, hi, adv, adv_lo, adv_hi, &
                 ! Make sure that our starting state is well-defined
                 dens_above = adv(domlo(1),j,k,URHO)
 
-                ! sometimes, we might be working in a corner
-                ! where the ghost cells above us have not yet
-                ! been initialized.  In that case, take the info
-                ! from the initial model
                 if (dens_above == ZERO) then
-                   x = problo(1) + delta(1)*(dble(domlo(1)) + HALF)
-
-                   call interpolate_sub(dens_above, x, idens_model)
-                   call interpolate_sub(temp_above, x, itemp_model)
-
-                   do m = 1, nspec
-                      call interpolate_sub(X_zone(m), x, ispec_model-1+m)
-                   end do
-
+                   ! sometimes, we might be working in a corner where
```

Comment:
This comment does not still make sense, I think we should remove this conditional logic entirely, our requirement is that we should always be integrating from a valid state, and if there's a combination where that's not true (like a corner with two intersecting inflow boundaries) then we should throw an error at initialization saying we don't support it.

Reply:
I think I've addressed this now